.drone.yml

```diff
@@ -84,6 +84,25 @@ pipeline:
       local: false
       event: [ push ]
 
+  update-search-index:
+    image: algolia/docsearch-scraper
+    entrypoint: ""
+    pull: true
+    secrets: [ 
+      algolia_app_id, algolia_config, 
+      algolia_index_name, algolia_search_api_key, 
+      algolia_write_api_key 
+    ]
+    environment:
+      - ALGOLIA_APP_ID=${ALGOLIA_APP_ID}
+      - ALGOLIA_API_KEY=${ALGOLIA_SEARCH_API_KEY}
+      - APPLICATION_ID=${ALGOLIA_APP_ID}
+      - API_KEY=${ALGOLIA_WRITE_API_KEY}
+      - ALGOLIA_INDEX_NAME=${ALGOLIA_INDEX_NAME}
+      - CONFIG=${ALGOLIA_CONFIG}
+    commands:
+      - pipenv run python -m src.index
+
```

Comment:
You should also add a `when` block, this should only be done on `push` event.

Reply:
Okay, I'll do a bit of reading to work out how to do that.